src/luascript.cpp

```diff
@@ -7327,19 +7327,32 @@ int LuaScriptInterface::luaCreatureGetPathTo(lua_State* L)
 
 int LuaScriptInterface::luaCreatureMove(lua_State* L)
 {
+	// creature:move(direction, tile[, flags = 0])
```

Comment:
There should be 2 signatures, `creature:move(direction)` and `creature:move(tile[, flags=0])` , not a single.